frontend/packages/dev-console/src/components/HintBlock.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import { PageHeading } from '@console/internal/components/utils';
+import SectionDivider from './import/section/FormSectionDivider';
+
+interface HintBlockProps {
+  title: string;
+  children?: React.ReactNode;
+}
+const HintBlock: React.FC<HintBlockProps> = (props) => (
```

Comment:
This component is used in a single location: DefaultPage.
Could you just merge the contents of this into DefaultPage? It doesn't really add much value over the PageHeading and SectionDivider separately.

Reply:
fixed it